investor_wallet_platform_base/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     "depends": [
         "cooperator",
         "easy_my_coop_loan",
-        "easy_my_coop_api",
+        "cooperator_api",
```

Comment:
@carmenbianca do not forget to mind for dependencies in other repositories (mainly this one and obeesdoo)